xbmc/interfaces/python/PythonInvoker.h

```diff
@@ -9,16 +9,10 @@
 #pragma once
 
 #include "interfaces/generic/ILanguageInvoker.h"
-#include "interfaces/legacy/Addon.h"
-#include "interfaces/python/LanguageHook.h"
 #include "threads/CriticalSection.h"
 #include "threads/Event.h"
 
-#include <map>
-#include <string>
-#include <vector>
```

Comment:
removing this (three) includes is a rebase error

Reply:
Should be resolved with the latest version